packages/eslint-plugin/package.json

```diff
@@ -66,6 +70,9 @@
         "@typescript-eslint/parser": {
             "optional": true
         },
+        "eslint": {
```

Comment:
ESlint should be considered optional. There is no way someone uses this package and doesnt have eslint isntalled

Reply:
This is for monorepo setup.

The reason being that ESLint is only required to be installed at the root of the monorepo not in individual packages. While the shared configurations must be installed in every packages.

We don't want every packages to install ESLint.